lib/nativescript-cli.ts

```diff
@@ -14,6 +14,9 @@ fiber(() => {
 
 	let commandDispatcher: ICommandDispatcher = $injector.resolve("commandDispatcher");
 
+	let messages = <IMessagesService>$injector.resolve("$messagesService");
```

Comment:
You may optionally use `let messages: IMessagesService = $injector.resolve... ` for less code clutter.